### DISCUSSION TITLE
Table hover usage

### DISCUSSION THREAD

**Discussion by itsNiccoloSabatini** [emilk/egui#4371] [Q&A]
hi, which is the best way to use the hover feature with TableBuilder in egui_extras? i need to create a mutable table that goes from 50 to 200 rows, how can i use the hover feat to make things appear in a side panel for each row?

**Comment by thmxv** [comment]
The [egui.rs](https://www.egui.rs/) table demo as the code provided: https://github.com/emilk/egui/blob/master/crates/egui_demo_lib/src/demo/table_demo.rs 

In this demo the row `Response` is used to select the row when it is clicked [line 269](https://github.com/emilk/egui/blob/0bc59f578b907e357f118dba362d89d5e36f4c31/crates/egui_demo_lib/src/demo/table_demo.rs#L269). But you can do the equivalent except that you can set the active row (the one to display the info for in the side panel) to the row index if the `Response` is [hovered](https://docs.rs/egui/0.27.2/egui/struct.Response.html#method.hovered).

Note: I have a hard time understanding why use hovered and not clicked. I think the usability would be better if you set the selection when the row is clicked and display the info in the side panel for the selected row. But this is specific to your app and its usability which I do not know about, so this is a bit unrelated.
